index.html

```diff
@@ -413,16 +413,58 @@ <h2>
         </ul>
         <p>
           Prior to presenting an <a>automated install prompt</a>, a user agent
-          MUST run the <a>steps to notify that an install prompt is
+          MAY run the <a>steps to notify that an install prompt is
           available</a>, to give the site the opportunity to prevent the
           default action (which is to install the application). Alternatively,
           the user agent MAY run the <a>steps to notify that an install prompt
           is available</a> at any time, giving the site the opportunity to show
           a <a>site-triggered install prompt</a> without automatically showing
-          the prompt.
+          the prompt. And, the user agent MAY <a>present an install prompt</a>
+          at any time, without giving the site an opportunity to prevent it (in
+          this case, it is RECOMMENDED that the automated prompt be minimally
```

Comment:
be -> is?

Reply:
Technically "be" is correct. Given it's hard to read this sentence, probably best to rephrase as "the automated prompt SHOULD be minimally..."

But anyway, I'm just throwing this up as a strawman proposal. I don't intend to commit this in its current form. Just collecting feedback on the proposal.